__tests__/AccountSettings.js

```diff
@@ -0,0 +1,77 @@
+import {
+  AWSInfoResponse,
+  getPersistedMockCall,
+  userResponse,
+} from 'testUtils/mockHttpCalls';
+import { renderRouteWithStore } from 'testUtils/renderUtils';
+
+const elementIDs = {
+  ChangeEmailForm: 'account-settings/change-email',
+  ChangePasswordForm: 'account-settings/change-password',
+  DeleteAccount: 'account-settings/delete-account',
+
+  EmailLabel: 'account-settings/email-label',
+  EmailDescription: 'account-settings/email-description',
+};
+
+const requests = {};
+
+describe('AccountSettings', () => {
+  beforeAll(() => {
+    requests.user = getPersistedMockCall('/v4/user/', userResponse);
+    requests.info = getPersistedMockCall('/v4/info/', AWSInfoResponse);
+    requests.orgs = getPersistedMockCall('/v4/organizations/');
+    requests.clusters = getPersistedMockCall('/v4/clusters/');
+    requests.catalogs = getPersistedMockCall('/v4/appcatalogs/');
+  });
+
+  afterAll(() => {
+    for (const req of Object.values(requests)) {
+      req.persist(false);
+    }
+  });
+
+  it('renders the account settings component on the "/account-settings" route', async () => {
+    const { findByText } = renderRouteWithStore('/account-settings', {});
+
+    await findByText('Your Account Settings');
+  });
+
+  it('renders the email change form', async () => {
+    const { findByTestId } = renderRouteWithStore('/account-settings', {});
+
+    const emailChangeForm = await findByTestId(elementIDs.ChangeEmailForm);
+
+    expect(emailChangeForm).not.toBeNull();
```

Comment:
And then there is no need to make those extra `expect`s.

Reply:
There is no need for the `expect`s anyway, because `findByTestId` throws an error if the element is not found. I added them because it's easier for us to go through and understand the tests